awsapilib/console/console.py

```diff
@@ -499,8 +502,12 @@ def _get_root_console_redirect(self, email, password, session, mfa_serial=None):
                               f'with status code: {response.status_code}')
         oidc = self._get_oidc_info(response.history[0].headers.get('Location'))
         response = self._resolve_account_type_response(email, session=session)
+        captcha_url = response.json().get('properties', {}).get('CaptchaURL')
```

Comment:
Shall we move these before lines 459 so we can use the variable there too?

Reply:
This lines are in different methods! So, there is not much of gain in it!

Reply:
😅 sorry, should have expanded the page !